### PR TITLE
added read time , views and claps

### PR DESCRIPTION
**Pull request by shivalee12**

### CODE REVIEW

File: blogs/models.py [hunk 1]
```diff
@@ -3,40 +3,55 @@
...
 
 
 class Blog(models.Model):
+    # fields for blogs form
     title = models.CharField(max_length=255)
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     title_image = models.ImageField(
         upload_to='blog/',
-        help_text='This image will be displayed as the title image of the blog.'
+        # help_text='This image will be displayed as\
```
Comment (0xhrsh) [inline blogs/models.py]: why is this commented?

File: blogs/models.py [hunk 2]
```diff
@@ -3,40 +3,55 @@
...
     )
     short_description = models.TextField()
     blog_content = RichTextUploadingField()
     published_date = models.DateField(auto_now_add=True)
     slug = models.SlugField(unique=True, blank=True)
     blog_views = models.IntegerField(default=0)
     blog_likes = models.ManyToManyField(User, related_name='likes')
+    read_time = models.TimeField(null=True, blank=True)
+
+    # def __str__(self):
```
Comment (0xhrsh) [inline blogs/models.py]: why is this commented?

File: blogs/views.py [hunk 3]
```diff
@@ -1,13 +1,17 @@
 from django.views.generic import ListView, DetailView, RedirectView
 from django.views.generic import CreateView, UpdateView, DeleteView
 from .models import Blog
-from django.urls import reverse_lazy, reverse
-from django.shortcuts import render, redirect, get_object_or_404
-from django.http import HttpResponseRedirect
-from django.contrib.auth import login, logout
+from django.urls import reverse_lazy
+from django.shortcuts import redirect, get_object_or_404
+# from django.http import HttpResponseRedirect
```
Comment (0xhrsh) [inline blogs/views.py]: remove this rather than commenting it

File: blogs/views.py [hunk 4]
```diff
@@ -17,14 +21,25 @@ class HomeView(ListView):
...
+        obj = get_object_or_404(Blog, slug=slug)
+        liked_user_list = []
+        for i in obj.blog_likes.all():
+            liked_user_list.append(i.id)
+
+        context['liked_user'] = liked_user_list
+        return context
+
 
 class BlogLikeToggle(RedirectView):
```
Comment (0xhrsh) [inline blogs/views.py:43]: are you using ```BlogLikeToggle``` or ```BlogLikeAPIToggle``` or both?

File: templates/details.html [hunk 5]
```diff
@@ -20,29 +67,31 @@ <h1>{{ blog.title }}</h1>
 <a href="{% url 'update' blog.slug %}"><button class="btn btn-secondary">Edit</button></a>
 <a href="{% url 'delete' blog.slug %}"><button class="btn btn-danger">Delete</button></a>
 <br /><br />
-
+<strong class="text-secondary"> {{ blog.number_of_likes }} Clap{{ blog.number_of_likes|pluralize }}</strong>
```
Comment (0xhrsh) [inline templates/details.html]: Have consistent nomenclature, either keep likes or keep claps

File: .gitignore [hunk 6]
```diff
@@ -114,6 +114,7 @@ celerybeat.pid
 
 # Environments
 .env
+.env.example
```
Comment (0xhrsh) [inline .gitignore]: you should actually change this to ```.env.*```. gitignore can have regex matches. read what regex is and what * means.

Comment (0xhrsh) [reply .gitignore]: Also, keep better names for the files. ```.example``` doesn't tell me what the file contains. Keep better names.

**Review by 0xhrsh — CHANGES_REQUESTED** [review]
Complete all 7 changes, also, make smaller PRs!

File: blogs/views.py (hunk 4 above)
Comment (shivalee12) [reply blogs/views.py:43]: we are using both

**Review by 0xhrsh — APPROVED** [review]

**Pull request merged**